configFileProductions/spring2024ConfigFiles/nunuH_Hxx/LCFIJetEnabled/whizardddsimgaudi.conf

```
@@ -0,0 +1,43 @@
+[whizard2]
+Version = 2.8.3
+
+[ddsim]
+SteeringFile = cld_steer.py
+ExtraCLIArguments = --vertexSigma 0.0098 2.54e-5 0.646 0.0063 --physics.rejectPDGs 1,2,3,4,5,6,12,14,16,21,23,24,25
+
+[gaudiapp]
+ExecutableName = k4run
+SteeringFile = CLDReconstruction.py
+InputFileFlag = --inputFiles
+OutputFileFlag = --Output_REC.filename
+Arguments = --enableLCFIJet
+
+[Production Parameters]
+machine = ee
+prodGroup = fcc_ee_spring2024_HiggsCouplings_LCFIJet_BeamSpotDec2022
+
+softwareVersion = key4hep_241128
+generatorApplication = whizard2
+simulationApplication = ddsim
+reconstructionApplication = gaudiapp
+generatorSteeringFile = wzp6_ee_nunuH_Htautau_ecm240.sin, wzp6_ee_nunuH_Hgg_ecm240.sin, wzp6_ee_nunuH_Hdd_ecm240.sin, wzp6_ee_nunuH_Huu_ecm240.sin, wzp6_ee_nunuH_Hss_ecm240.sin, wzp6_ee_nunuH_Hcc_ecm240.sin, wzp6_ee_nunuH_Hbb_ecm240.sin
+
+configVersion = key4hep-devel-4
+configPackage = fccConfig
+eventsPerJobs = 1000, 1000, 1000, 1000, 1000, 1000, 1000
+
+numberOfTasks = 1, 1, 1, 1, 1, 1, 1
+
+campaign = test_spring2024
+energies = 240, 240, 240, 240, 240, 240, 240
+processes = Htautau, Hgg, Hdd, Huu, Hss, Hcc, Hbb
```

```suggestion
processes = Htautau, Hgg, Hdd, Huu, Hss, Hcc, Hbb
ProdIDs = 16764,16767,16770,16773,16776,16779,16782
```
And changing `ProdTypes = Rec` let's one reuse the existing simulation files.

---

Good point.